src/components/Popup.js

```diff
@@ -1,10 +1,12 @@
-// responsible for opening and closing all the different pop ups
 export default class Popup {
   constructor({ popupSelector }) {
     this._popupElement = document.querySelector(popupSelector);
     this._closeButton = this._popupElement.querySelector(
       ".modal__close-button"
     );
+    this._buttonElement = this._popupElement.querySelector(
```

Comment:
You shouldn't find a submit button in `Popup` because the image modal doesn't have it